.github/workflows/pr-check-changeset.yml

```diff
@@ -51,6 +51,17 @@ jobs:
           # JSON output is piped through jq to compact it to a single line
           pnpm exec flub check changeset --branch=${{ github.base_ref }} --json | jq -c > changeset-metadata.json
 
+      # Sets required = true/false based on the changeset-required label on the PR
+      - name: Changeset required
+        if: contains(github.event.pull_request.labels.*.name, 'changeset-required')
+        run: |
+          echo $(jq -c '. += {required: true}' changeset-metadata.json) > changeset-metadata.json
+
+      - name: Changeset not required
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'changeset-required') }}
```

Comment:
We need to wrap it just for the negation? Ugh, come on Github...

Reply:
I think it's yaml, not GH in particular. Using `""` might also work, but this is more explicit.